### ISSUE TITLE
Problem selecting feed while showing only unread feeds

### ISSUE THREAD

**Issue by brentsimmons** [Ranchero-Software/NetNewsWire#1710]
Quote: “I quit NNW, and restarted. I have the “Refresh to clear feeds” selected, but that didn’t happen, and the New Scientist feed won’t show the supposed item it is displaying no matter how many times I tap on it.”

![screenshot_01](https://user-images.githubusercontent.com/1297121/73130063-659b0280-3fa6-11ea-8b74-5fb1456d5151.jpg)

**Comment by vincode-io** [comment]
It looks like the app launched and restored an article selection.  It will show the article in the timeline even if the filter is on and the article is read.  We do this intentionally so that the UI is consistent.  I think this person is just confused as to why that article is there in the timeline.

As to how the New Scientist feed could be unelectable, I've got nothing yet.  This is the first I've heard of that kind of bug.  I tried recreating the above scenario and was able to recreate the same screenshot, but was able to select feeds afterwards.
